### PR TITLE
Arcconf Plugin patch: Fix for 'disks' function

### PR DESCRIPTION
**Pull request by sharath-t**
issue: column 'Name' has a string prefix 'test'
		fix : replace the 'test' string with manufacturer name
		
issue: column 'vdp' and disk path is empty. no logic implemented
		fix: for the Drives having 'physicalDriveName', use the lsm native function 'LocalDisk.vpd83_get(disk_path)' to get the vdp info
		
issue: column 'Size' showing incorrect data
		fix: code was looking for 'physicalBlockSize' where this was different than 'blockSize'. changed the code to fetch 'blockSize'
		
issue: column 'Link Type' was displaying incorrect information
		fix: code was using the same native Disk constants to represent Disk Type and Link Type. implemented separate functions for Type and Link type

### CODE REVIEW

File: plugin/arcconf/arcconf.py [hunk 1]
```diff
@@ -541,7 +539,7 @@ def disks(self, search_key=None, search_value=None,
                                     arcconf_disk, sys_id, cntrl_num))
 
         return search_property(rc_lsm_disks, search_key, search_value)
-
+
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: Please remove this white space.

File: plugin/arcconf/arcconf.py [hunk 2]
```diff
@@ -484,14 +484,11 @@ def volumes(self, search_key=None, search_value=None,
...
+        disk_name = "%s - %s" % (arcconf_disk['vendor'], arcconf_disk['model'])
         disk_type = _disk_type_of(arcconf_disk)
-        link_type = disk_type
+        link_type = _disk_link_type_of(arcconf_disk)
 
-        try:
-            blk_size = int(arcconf_disk['physicalBlockSize'])
-        except KeyError:
-            blk_size = int(arcconf_disk['blockSize'])
+        blk_size = int(arcconf_disk['blockSize'])
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: @sharath-t Can you explain to me why `physicalBlockSize` is not used here?

File: plugin/arcconf/arcconf.py (hunk 1 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: Done - commit id - 6651667aa6ba35ac90226cfc320baa488fd94e90

File: plugin/arcconf/arcconf.py (hunk 2 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: Thanks for the catch.
i discussed internally and found that 'physicalBlockSize' was added to support 4kn drives. so i have reverted the code to earlier state.

**Comment by tasleson** [comment]
Integrated into https://github.com/libstorage/libstoragemgmt/pull/386

**Pull request closed without merging**